app/ante/fee.go

```diff
@@ -52,26 +51,23 @@ func ValidateTxFee(ctx sdk.Context, tx sdk.Tx, paramKeeper params.Keeper) (sdk.C
 	}
 
 	// Ensure that the provided fee meets a network minimum threshold.
-	// Network minimum fee only applies to app versions greater than one.
-	if ctx.BlockHeader().Version.App > v1.Version {
```

Comment:
Funnily removing this code caused a side effect in tests below as minfee was kicking in a failing txs